modules/processing/redshift/doc/ST_DELAUNAYLINES.md

```diff
@@ -0,0 +1,22 @@
+### ST_DELAUNAYLINES
+
+{{% bannerNote type="code" %}}
+processing.ST_DELAUNAYLINES(points)
+{{%/ bannerNote %}}
+
+**Description**
+
+Calculates the Delaunay triangulation of the points provided. A MultiLineString object is returned.
+
+* `points`: `GEOMETRY` MultiPoint input to the Delaunay triangulation.
+
+**Return type**
+
+`VARCHAR(MAX)`
```

Comment:
We could return a `SUPER` here and also in voronoi functions by just doing a json_parse, so hopefully in the future (unless they implement new types) we don't have to readapt functions.

Reply:
They sent us a solution on how to use `SUPER` types so it's the way to go. I'm still testing as they say that we have to add extra paths if we want them to contain schemas.